src/pages/signin/LoginForm/BaseLoginForm.tsx

```diff
@@ -307,23 +274,24 @@ function LoginForm(props) {
                     maxLength={CONST.LOGIN_CHARACTER_LIMIT}
                 />
             </View>
-            {!_.isEmpty(props.account.success) && <Text style={[styles.formSuccess]}>{props.account.success}</Text>}
-            {!_.isEmpty(props.closeAccount.success || props.account.message) && (
+            {account?.success && <Text style={[styles.formSuccess]}>{account.success}</Text>}
+            {(closeAccount?.success ?? account?.message) && (
                 // DotIndicatorMessage mostly expects onyxData errors, so we need to mock an object so that the messages looks similar to prop.account.errors
                 <DotIndicatorMessage
                     style={[styles.mv2]}
                     type="success"
-                    messages={{0: props.closeAccount.success || props.account.message}}
+                    // eslint-disable-next-line @typescript-eslint/naming-convention
+                    messages={{0: (closeAccount?.success ?? account?.message ?? '') as MaybePhraseKey}}
```

Comment:
Use `||` instead of `??`. Is there a reason to cast to `MaybePhraseKey`?

Reply:
If `closeAccount?.success` is empty string we want to use `account?.message` as it may have an actual string.